<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="3" ignored="3" total="11" passed="4" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Campagin Expo_622 added successfully]]>
    </line>
    <line>
      <![CDATA[Campagin Expo_1068 added successfully]]>
    </line>
    <line>
      <![CDATA[Product Laptop_5427 added successfully]]>
    </line>
    <line>
      <![CDATA[Contact Smith_643 added successfully]]>
    </line>
  </reporter-output>
  <suite started-at="2025-03-29T20:52:54 IST" name="Suite" finished-at="2025-03-29T20:54:57 IST" duration-ms="122778">
    <groups>
      <group name="IntegrationTest">
        <method signature="CreateProduct.createProductTest()[pri:0, instance:ProductTest.CreateProduct@5aebe890]" name="createProductTest" class="ProductTest.CreateProduct"/>
      </group> <!-- IntegrationTest -->
      <group name="RegressionTest">
        <method signature="CreateContact.createContactWithCampaginTest()[pri:0, instance:ContactTest.CreateContact@65d09a04]" name="createContactWithCampaginTest" class="ContactTest.CreateContact"/>
      </group> <!-- RegressionTest -->
    </groups>
    <test started-at="2025-03-29T20:52:54 IST" name="Test" finished-at="2025-03-29T20:54:57 IST" duration-ms="122778">
      <class name="LoginTest.LoginTest">
        <test-method is-config="true" signature="beforeSuite()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:52:54 IST" name="beforeSuite" finished-at="2025-03-29T20:52:54 IST" duration-ms="860" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:52:55 IST" name="beforeTest" finished-at="2025-03-29T20:52:55 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:52:55 IST" name="beforeClass" finished-at="2025-03-29T20:52:56 IST" duration-ms="1711" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:52:56 IST" name="beforeMethod" finished-at="2025-03-29T20:52:58 IST" duration-ms="2122" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method retried="true" signature="loginTest()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:52:58 IST" name="loginTest" finished-at="2025-03-29T20:53:01 IST" duration-ms="2142" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Validation is failed expected [http://49.249.28.218:8098/dashboar] but found [http://49.249.28.218:8098/dashboard]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Validation is failed expected [http://49.249.28.218:8098/dashboar] but found [http://49.249.28.218:8098/dashboard]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at LoginTest.LoginTest.loginTest(LoginTest.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:53:01 IST" name="afterMethod" finished-at="2025-03-29T20:53:03 IST" duration-ms="2800" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:53:03 IST" name="beforeMethod" finished-at="2025-03-29T20:53:05 IST" duration-ms="1393" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method retried="true" signature="loginTest()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:53:05 IST" name="loginTest" finished-at="2025-03-29T20:53:07 IST" duration-ms="2040" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Validation is failed expected [http://49.249.28.218:8098/dashboar] but found [http://49.249.28.218:8098/dashboard]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Validation is failed expected [http://49.249.28.218:8098/dashboar] but found [http://49.249.28.218:8098/dashboard]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at LoginTest.LoginTest.loginTest(LoginTest.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:53:07 IST" name="afterMethod" finished-at="2025-03-29T20:53:17 IST" duration-ms="10187" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:53:17 IST" name="beforeMethod" finished-at="2025-03-29T20:53:22 IST" duration-ms="5005" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method retried="true" signature="loginTest()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:53:22 IST" name="loginTest" finished-at="2025-03-29T20:53:22 IST" duration-ms="472" status="SKIP">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[unknown error: failed to change window state to 'normal', current state is 'maximized'
  (Session info: chrome=134.0.6998.178)
Build info: version: '4.26.0', revision: '8ccf0219d7'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [47c1f8f7887fda74a94ecd210af4f843, maximizeCurrentWindow {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 134.0.6998.178, chrome: {chromedriverVersion: 134.0.6998.165 (fd886e2cb29..., userDataDir: C:\Users\Admin\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52398}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52398/devtoo..., se:cdpVersion: 134.0.6998.178, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 47c1f8f7887fda74a94ecd210af4f843]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: unknown error: failed to change window state to 'normal', current state is 'maximized'
  (Session info: chrome=134.0.6998.178)
Build info: version: '4.26.0', revision: '8ccf0219d7'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [47c1f8f7887fda74a94ecd210af4f843, maximizeCurrentWindow {}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 134.0.6998.178, chrome: {chromedriverVersion: 134.0.6998.165 (fd886e2cb29..., userDataDir: C:\Users\Admin\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52398}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52398/devtoo..., se:cdpVersion: 134.0.6998.178, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 47c1f8f7887fda74a94ecd210af4f843
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:501)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:485)
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:618)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:622)
at org.openqa.selenium.remote.RemoteWebDriver$RemoteWebDriverOptions$RemoteWindow.maximize(RemoteWebDriver.java:1061)
at LoginTest.LoginTest.loginTest(LoginTest.java:23)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:53:22 IST" name="afterMethod" finished-at="2025-03-29T20:53:28 IST" duration-ms="5274" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:53:28 IST" name="beforeMethod" finished-at="2025-03-29T20:53:29 IST" duration-ms="1375" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="loginTest()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:53:29 IST" name="loginTest" finished-at="2025-03-29T20:53:31 IST" duration-ms="2041" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Validation is failed expected [http://49.249.28.218:8098/dashboar] but found [http://49.249.28.218:8098/dashboard]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Validation is failed expected [http://49.249.28.218:8098/dashboar] but found [http://49.249.28.218:8098/dashboard]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at LoginTest.LoginTest.loginTest(LoginTest.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:53:31 IST" name="afterMethod" finished-at="2025-03-29T20:53:37 IST" duration-ms="5218" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:53:37 IST" name="afterClass" finished-at="2025-03-29T20:53:37 IST" duration-ms="374" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:54:57 IST" name="afterTest" finished-at="2025-03-29T20:54:57 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method is-config="true" signature="afterSuite()[pri:0, instance:LoginTest.LoginTest@1e66f1f5]" started-at="2025-03-29T20:54:58 IST" name="afterSuite" finished-at="2025-03-29T20:54:58 IST" duration-ms="8" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- LoginTest.LoginTest -->
      <class name="ProductTest.CreateProduct">
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:ProductTest.CreateProduct@5aebe890]" started-at="2025-03-29T20:54:11 IST" name="beforeClass" finished-at="2025-03-29T20:54:12 IST" duration-ms="1009" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:ProductTest.CreateProduct@5aebe890]" started-at="2025-03-29T20:54:12 IST" name="beforeMethod" finished-at="2025-03-29T20:54:14 IST" duration-ms="1992" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="createProductTest()[pri:0, instance:ProductTest.CreateProduct@5aebe890]" started-at="2025-03-29T20:54:14 IST" name="createProductTest" finished-at="2025-03-29T20:54:27 IST" duration-ms="13381" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Product Laptop_5427 added successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- createProductTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:ProductTest.CreateProduct@5aebe890]" started-at="2025-03-29T20:54:27 IST" name="afterMethod" finished-at="2025-03-29T20:54:28 IST" duration-ms="1090" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:ProductTest.CreateProduct@5aebe890]" started-at="2025-03-29T20:54:28 IST" name="afterClass" finished-at="2025-03-29T20:54:28 IST" duration-ms="287" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- ProductTest.CreateProduct -->
      <class name="CampaignTest.CreateCampaign">
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:CampaignTest.CreateCampaign@4961f6af]" started-at="2025-03-29T20:53:37 IST" name="beforeClass" finished-at="2025-03-29T20:53:38 IST" duration-ms="997" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:CampaignTest.CreateCampaign@4961f6af]" started-at="2025-03-29T20:53:38 IST" name="beforeMethod" finished-at="2025-03-29T20:53:40 IST" duration-ms="1722" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="createCampaginWithDateTest()[pri:0, instance:CampaignTest.CreateCampaign@4961f6af]" started-at="2025-03-29T20:53:40 IST" name="createCampaginWithDateTest" finished-at="2025-03-29T20:53:53 IST" duration-ms="13591" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Campagin Expo_622 added successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- createCampaginWithDateTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:CampaignTest.CreateCampaign@4961f6af]" started-at="2025-03-29T20:53:53 IST" name="afterMethod" finished-at="2025-03-29T20:53:55 IST" duration-ms="1234" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:CampaignTest.CreateCampaign@4961f6af]" started-at="2025-03-29T20:53:55 IST" name="beforeMethod" finished-at="2025-03-29T20:53:57 IST" duration-ms="2034" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="createCampaignTest()[pri:0, instance:CampaignTest.CreateCampaign@4961f6af]" started-at="2025-03-29T20:53:57 IST" name="createCampaignTest" finished-at="2025-03-29T20:54:09 IST" duration-ms="12002" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Campagin Expo_1068 added successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- createCampaignTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:CampaignTest.CreateCampaign@4961f6af]" started-at="2025-03-29T20:54:09 IST" name="afterMethod" finished-at="2025-03-29T20:54:10 IST" duration-ms="1683" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:CampaignTest.CreateCampaign@4961f6af]" started-at="2025-03-29T20:54:10 IST" name="afterClass" finished-at="2025-03-29T20:54:11 IST" duration-ms="334" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- CampaignTest.CreateCampaign -->
      <class name="ContactTest.CreateContact">
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:ContactTest.CreateContact@65d09a04]" started-at="2025-03-29T20:54:28 IST" name="beforeClass" finished-at="2025-03-29T20:54:30 IST" duration-ms="1046" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:ContactTest.CreateContact@65d09a04]" started-at="2025-03-29T20:54:30 IST" name="beforeMethod" finished-at="2025-03-29T20:54:32 IST" duration-ms="2514" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="createContactWithCampaginTest()[pri:0, instance:ContactTest.CreateContact@65d09a04]" started-at="2025-03-29T20:54:32 IST" name="createContactWithCampaginTest" finished-at="2025-03-29T20:54:55 IST" duration-ms="23365" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Contact Smith_643 added successfully]]>
            </line>
          </reporter-output>
        </test-method> <!-- createContactWithCampaginTest -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:ContactTest.CreateContact@65d09a04]" started-at="2025-03-29T20:54:55 IST" name="afterMethod" finished-at="2025-03-29T20:54:57 IST" duration-ms="1537" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:ContactTest.CreateContact@65d09a04]" started-at="2025-03-29T20:54:57 IST" name="afterClass" finished-at="2025-03-29T20:54:57 IST" duration-ms="330" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- ContactTest.CreateContact -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
